File: solutions/swift/raindrops/2/Sources/Raindrops/Raindrops.swift
func raindrops(_ number: Int) -> String {
  let ret =
    (number.isMultiple(of: 3) ? "Pling" : "")
    + (number.isMultiple(of: 5) ? "Plang" : "")
    + (number.isMultiple(of: 7) ? "Plong" : "")
  return ret.isEmpty ? String(number) : ret
}

File: solutions/cpp/raindrops/3/raindrops.cpp
#include "raindrops.h"
#include <array>

namespace raindrops {

    using namespace std;

    const array<pair<int, string>, 3> sounds = {{{3, "Pling"},
                                                 {5, "Plang"},
                                                 {7, "Plong"},
                                                }};

    string convert(int n) {
        string str;
        for (const auto &s: sounds) {
            if (n % s.first == 0)
                str += s.second;
        }
        return str.empty() ? to_string(n) : str;
    }

}  // namespace raindrops
